index.html

```diff
@@ -0,0 +1,27 @@
+<html lang="en">
+      <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="stylesheet.css">
+        <title>Personal Portfolio: Omar Gutierrez</title> 
+     </head>
+     <body>
+      <header class="header">   
+        <nav class="toolbar">
+          <p class="logo">Omar Gutierrez</p>
+          <div class="hamburger"><img src="./images/Hamburger.svg" alt="menu"></div>
+        </nav>
+        <section class="headline">
+          <h1 class="greeting">Hey There. I'm Omar.</h1>
+          <h3 class="presentation">I'm a Software Developer</h3>
+          <p class="description">I will help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesistate to contact me</p>
```

Comment:
- [ ] It is required to stick to the figma design as close as possible. This includes, margin, padding, font, typography among others. Kindly reduce the space between the header and the headline on your page so as to match the figma design

### Your Design
![your-title-spacing](https://github.com/ommarz/Personal_Portfolio/assets/26597432/5706757c-50ce-4bf8-8a45-662a2726994d)

### Figma Design
![figma-title-spacing](https://github.com/ommarz/Personal_Portfolio/assets/26597432/56ed4717-44c0-4963-8231-07efd17d9fc3)

- [ ] The same rule applies to the line-spacing between your main title, subtitle, and the paragraph. Kindly compare the two screenshots below and walk your design alongside the figma design

### Your Design
![your-line-spacing](https://github.com/ommarz/Personal_Portfolio/assets/26597432/507615d0-e8e0-451a-800a-806a001d63ff)

### Figma Design
![figma-line-spacing](https://github.com/ommarz/Personal_Portfolio/assets/26597432/02076db7-430e-4582-825c-2dcf9257fc5f)